apps/test/util/runTests.js

```diff
@@ -0,0 +1,68 @@
+var which = require('npm-which')(__dirname).sync;
+var mochify = require('mochify');
+var http = require('http');
+
+// TODO (brent) When we move to npm/gulp, we will want to take care of this
+// logic in our gulpfile instead
+// Right now we manually create a symbolic link to @cdo/apps
+var fs = require('fs');
+var exec = require('child_process').exec;
+var target = __dirname + '/../../src/';
+var nodePath = __dirname + '/../../node_modules/@cdo/apps';
+var command = '';
+if (!fs.existsSync(target)) {
+  command = 'ln -s ' + target + ' ' + nodePath;
+}
```

Comment:
Some reference notes on this issue:

The issue of making local `require` paths work better with single-repository, multi-package projects has been a limitation of node/npm for a long time, and the number of hacks, workarounds and partial fixes that the community has put forth is [ridiculously long and complicated](https://gist.github.com/branneman/8048520). The method here (using a manual symlink within node_modules) is the one recommended by Substack in the [browserify handbook](https://github.com/substack/browserify-handbook#avoiding-), but it also comes loaded with tons of baggage (breaks browserify transforms, breaks `npm prune`/`npm update`, requires manual symlink management for each dependency, special gitignore config needed to include specific files in `node_modules`, etc etc). The least-painful alternative I've found so far is to use a combination of the npm@2 [local paths](https://github.com/npm/npm/blob/master/doc/files/package.json.md#local-paths) feature and automatically [symlink local dependencies](https://github.com/npm/npm/issues/6251) using [linklocal](https://github.com/timoxley/linklocal), until the issue is (hopefully eventually) better resolved in a future npm version.

If this code works for now it's fine, but we should keep in mind that this is a long-standing, thorny issue that the nodejs community still hasn't fully worked out yet, so we should keep an eye out on any future developments that could make this easier to maintain in the future.